packages/core/types/src/file/provider.ts

```diff
@@ -128,6 +129,15 @@ export interface IFileProvider {
    */
   delete(fileData: ProviderDeleteFileDTO): Promise<void>
```

Comment:
thought/suggestion: Should we, instead, have the provider `delete` method accept `ProviderDeleteFileDTO | ProviderDeleteFileDTO[]` instead of adding a new method?

Reply:
+1, I also think this is a bit more inlign with how we usually define the public service interfaces